x-pack/plugins/security_solution/common/endpoint/generate_data.test.ts

```diff
@@ -113,6 +113,8 @@ describe('data generator', () => {
         percentWithRelated: 100,
         relatedEvents: 0,
         relatedAlerts: 0,
+        useAncestryArray: true,
```

Comment:
These changes are in this PR: https://github.com/elastic/kibana/pull/70129

they will be removed once the other PR gets merged.